pkg/mqtt5/bootstrap/handler.go

```diff
@@ -28,21 +28,14 @@ func Mqtt5BootstrapHandler(ctx context.Context, wg *sync.WaitGroup, startupTimer
 	}
 
 	// create client and connect
-	mutex := &sync.RWMutex{}
-	var clientMap Mqtt5ClientMap
-	if clientMap.mqtt5Clients == nil {
-		clientMap.mqtt5Clients = map[string]*mqtt5.Mqtt5Client{}
-		clientMap.mutex = mutex
-	}
-
+	clientMap := InitMqtt5ClientMap()
 	for configName, mqttConfig := range mqtt5ConfigMap {
-		if err := config.Validate(mqttConfig); err != nil {
+		if err := validator.Validate(mqttConfig); err != nil {
 			logger.Errorf("Mqtt5Config %s validation error: %s", configName, err)
 			return false
 		}
 
 		client := mqtt5.NewMqtt5Client(mqttConfig)
-		client.SetMutex(mutex)
 
 		if err := client.SetAuthData(secretProvider, logger); err != nil {
 			logger.Errorf("Setting MQTT 5 auth data failed: %v", err)
```

Comment:
after implementing the Put command in `Mqtt5ClientMap`, use it for line 45

Reply:
updated